app/views/play/menu/InventoryModal.coffee

```diff
@@ -76,7 +76,10 @@ module.exports = class InventoryModal extends ModalView
       programmableConfig = _.find(item.get('components'), (c) -> c.config?.programmableProperties)?.config
       item.programmableProperties = (programmableConfig?.programmableProperties or []).concat programmableConfig?.moreProgrammableProperties or []
     @itemsProgrammablePropertiesConfigured = true
-    @equipment = @options.equipment or @options.session?.get('heroConfig')?.inventory or me.get('heroConfig')?.inventory or {}
+    if me.isStudent()
+      @equipment = me.get('heroConfig')?.inventory or @options.equipment or @options.session?.get('heroConfig')?.inventory or {}
+    else
+      @equipment = @options.equipment or @options.session?.get('heroConfig')?.inventory or me.get('heroConfig')?.inventory or {}
```

Comment:
Old behavior would be: "If there is a specific equipment for that level use that, otherwise if the user has equipped something for that level use that, otherwise use the player's last gear, otherwise start blank."

Well, since gear is cosmetic in the game, it doesn't matter that we store the gear in the session, but instead make it global carried between each level, which becomes: "If the player has equipped gear use that, otherwise use the specific level's gear, otherwise use the past session's gear, otherwise empty."